src/utils.js

```diff
@@ -324,6 +324,70 @@ go.utils = {
     },
 
 
+// STAGE-BASE HELPERS
+
+    read_subscription: function(im, subscription_id) {
```

Comment:
`read_` seems out-of-place. We've used `get_` elsewhere, which I think is more appropriate.